charts/logan/values.yaml

```diff
@@ -45,12 +45,12 @@ resourceNamePrefix: "{{ .Values.global.resourceNamePrefix }}"
 # --  Kubernetes ServiceAccount
 serviceAccount: "{{ .Values.global.resourceNamePrefix }}"
 image:
-   # Image pull secrets for. Secret must be in the namespace defined by namespace
-   imagePullSecrets:
    # -- Replace this value with actual docker image url
-   url: container-registry.oracle.com/oci_observability_management/oci-la-fluentd-collector:1.0.0
+   imageUrl: container-registry.oracle.com/oci_observability_management/oci-la-fluentd-collector:1.0.0
```

Comment:
url seems to be fine. Don't want to introduce breaking change on this important variable.

Reply:
thanks, fixed (reverted) to `url` to avoid breaking change